src/main/java/serguei/http/HttpHeaders.java

```diff
@@ -25,6 +26,7 @@
     private static final int MAX_HEADER_NUMBER = 1000;
     private static final int UPPER_LOW_DIFF = 'a' - 'A';
     private static final BodyEncoding NO_BODY_ENCODING = new BodyEncoding(false, null);
+    public static final String MAX_FILE_DOWNLOAD = "X-Max-File-Download";
```

Comment:
This library so far did not use any non-standard headers. This introduces unnecessary functionality.